go/libraries/doltcore/env/actions/diffs.go

```diff
@@ -104,3 +120,66 @@ func GetTableDiffs(ctx context.Context, dEnv *env.DoltEnv) (*TableDiffs, *TableD
 
 	return stagedDiffs, notStagedDiffs, nil
 }
+
+func NewDocDiffs(ctx context.Context, newerLicenseBytes, newReadmeBytes []byte, older *doltdb.RootValue) (*DocDiffs, error) {
```

Comment:
might want to modify function signature so that you can add new doc types in the future without having to change the function signature for each new type of document.

Reply:
Changed `NewDocDiffs` to take in a slice of `DocDetails`: https://github.com/liquidata-inc/dolt/pull/222/files#diff-0696625847c6b30eef8d68a0543236a1R124

...so now `rootVal.DocDiff` also takes in a slice of DocDetails, and can process any number of docs: https://github.com/liquidata-inc/dolt/pull/222/files#diff-6f09c9d7a33cd9d8093aa6d760e03081R490